source/features/hide-navigation-hover-highlight.tsx

```diff
@@ -20,5 +11,12 @@ function init() {
 
 features.add({
 	id: 'hide-navigation-hover-highlight',
+	description: `Some lists like notifications, file lists, and issue lists,
+			are highlighted as you move the mouse over them. This highlight
+			is useful when navigating via the keyboard (j/k), but annoying
+			when just moving the mouse around.
+			This feature will hide the highlight until the first keyboard
+			navigation, then it will be displayed until the next full reload.
```

Comment:
Don't hard-wrap.

Reply:
We should still leave the double linebreak between “paragraphs,” right? A-la markdown